src/controllers/screen.js

```diff
@@ -0,0 +1,15 @@
+const { createScreenDb } = require("../domains/screen");
+
+const createAnotherScreen = async (req, res) => {
+  const { number } = req.body;
+  try {
+const createdScreen = await createScreenDb ( number )
+res.status(201).json({
+    screen: createdScreen, 
+})
+  } catch (error) {
+    throw error
+  }
+};
```

Comment:
This try / catch is unnecessary because you're not doing anything with the error when it's been caught